apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.ts

```diff
@@ -370,9 +370,9 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
         private salesPersonStore: SalesPersonComponentStore
     ) {
         this.breakPointObservar.observe([
-            '(max-width: 991px)'
+            BREAKPOINT_SCREEN_SIZE.TAB
         ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
-            this.isTabScreen = result.matches;
+            this.isTabScreen = result.breakpoints[BREAKPOINT_SCREEN_SIZE.TAB];
```

Comment:
```suggestion
            this.isTabScreen = result?.breakpoints[BREAKPOINT_SCREEN_SIZE.TAB];
```

do every where, when required